lib/src/scheduler.dart

```diff
@@ -0,0 +1,42 @@
+part of '../cron.dart';
+
+/// The cron schedule.
+class Schedule {
```

Comment:
I can see that some  parts of the library can be moved out, but I see very little reason to do it for all small class. Please let's not move everything around, as it is hard to assess the actual diff of the change.